http/1.5/modules/ROOT/nav.adoc

```diff
@@ -1,24 +1,25 @@
 .xref:index.adoc[HTTP Connector]
 * xref:index.adoc[About HTTP Connector]
-* xref:http-about-http-request.adoc[HTTP Requests and Responses]
-* xref:http-authentication.adoc[Authenticate HTTP Requests]
+** xref:http-listener-ref.adoc[HTTP Listener Configuration Reference]
+** xref:http-request-ref.adoc[HTTP Request Configuration Reference]
+** xref:http-about-http-request.adoc[HTTP Requests and Responses]
+** xref:http-authentication.adoc[Authenticate HTTP Requests]
+** xref:http-about-loading-static-resource.adoc[About Loading a Static Resource]
+** xref:http-about-http-connector-migration.adoc[HTTP Connector Migration]
 * xref:http-about-http-connector.adoc[HTTP Connector (Design Center)]
```

Comment:
I would do away completely with the Design Center topic unless there is some valuable info in there (not just the generic configuring in Design Center topic).

Reply:
Yes, I'm planning that for the second phase of updating the info on each page. I want to make sure that I'm not removing valuable content. Therefore, I would like first to reorganize the ToC